README.md

```diff
@@ -15,5 +15,5 @@ For more information and more applications see https://flathub.org
 
 Contributing to Flathub
 -----------------------
-
```

Comment:
This should be removed. Normally you should open PR against `new-pr` repository.